nbs/examples/ElectricityLoadForecasting.ipynb

```diff
@@ -0,0 +1,3625 @@
+{
```

Comment:
<p>Typo: We will leave the last 24 observations (the last day)<s>2</s> as the test set<strong>, so ... </strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>